lightwood/encoder/categorical/autoencoder.py

```diff
@@ -101,12 +100,11 @@ def encode(self, column_data):
                 embeddings = self.encoder(oh_encoded_tensor)
                 return embeddings.to('cpu')
 
-    def decode(self, encoded_data):
-        self.onehot_encoder.predict_proba = self.predict_proba
+    def decode(self, encoded_data, predict_proba=False):
```

Comment:
this feels hacky, I'd prefer to skip the encoder altogether. At *worst* this should be called something like `get_raw_logits` or something like that, since it's basically just "give me the original one-hot vector".